tests/default_vars.sh

```diff
@@ -1000,3 +1008,103 @@ export TFREEZE_OPTION='linear_salt'
 export BL_SUFFIX=""
 export RT_SUFFIX=""
 }
+export_datm_cdeps_hafs ()
```

Comment:
can we use export_datm_cdeps for export_datm_cdeps_hafs? I see both S2S and HAFS is turned off in export_datm_cdeps, maybe one of them can be turned on in the test. What are the differences between export_datm_cdeps and export_datm_cdeps_hafs?

Reply:
Currently these hafs related export sections have been renamed into export_hafs_*. For the unification with other export sections, we will do that in the future when further unifying/cleaning up all HAFS related regression tests.